src/main/scala/com/atomist/rug/runtime/js/migrations/BringUpToDate.scala

```diff
@@ -0,0 +1,10 @@
+package com.atomist.rug.runtime.js.migrations
+
+/**
+  * Keep this migration current to bring rugs up to date.
+  */
+object BringUpToDate {
+
+  val Migration: Migration = NodeNameToProperty andThen NodeTagsToProperty
```

Comment:
concern: This will run on all rugs forever. It's quite feasible that someday someone will have a function nodeName() somewhere. This will change it, and that's confusing.

Let's check the version of TS Modules they have, and only run this migration if they're out of date.